cli/tests/integration/vendor_tests.rs

```diff
@@ -0,0 +1,331 @@
+// Copyright 2018-2022 the Deno authors. All rights reserved. MIT license.
```

Comment:
Most of the tests aren't in this file. They'll be in `cli/tools/vendor/build.rs`. This is just the final integration tests.